src/Google/Client.php

```diff
@@ -366,7 +366,7 @@ public function authorize(ClientInterface $http = null)
     } elseif ($token = $this->getAccessToken()) {
       $scopes = $this->prepareScopes();
       // add refresh subscriber to request a new token
-      if ($this->isAccessTokenExpired() && isset($token['refresh_token'])) {
+      if (isset($token['refresh_token']) && $this->isAccessTokenExpired()) {
```

Comment:
`isset()` faster, e.g. Symfony used it for some performance optimization, so we just put "faster" to fail part of condition at first place. Micro-optimization I would say.